### PR TITLE
purge: unused and excess components

### PR DESCRIPTION
**Pull request by BunsDev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,22 +15,19 @@
...
 - [node](https://nodejs.org/en/download/)
-  - You'll know you did it right if you can run `node --version` and you see a response like `v16.13.0`  
-- [npm](https://www.npmjs.com/get-npm)
-  - You'll know you did it right if you can run `npm --version` and you see a response like `8.1.0`
+  - You'll know you did it right if you can run `node --version` and you see a response like `v16.13.0`
 - [deno](https://docs.deno.com/runtime/manual/getting_started/installation)
   - You'll know you did it right if you can run `deno --version` and you see a response like `deno 1.40.5 (release, x86_64-apple-darwin) v8 12.1.285.27 typescript 5.3.3`
 
-## Installation
+## Apps
```
Comment (zeuslawyer) [inline README.md:22]: Please make the top level readme immediately usable by a first time visitors.  Eg point out that each app dir has its own README and instructons are there.

**Review by zeuslawyer — DISMISSED** [review]
Too lagguy to review fully. out of time so merge and hope the demo goes well.

**Review by pappas999 — DISMISSED** [review]
LGTM

**Review by rgottleber — APPROVED** [review]
🤷‍♂️LGTM

**Review by rgottleber — APPROVED** [review]

**Review by disbitski-cll — APPROVED** [review]
LGTM!

**Pull request merged**